eng/build.ps1

```diff
@@ -723,6 +723,30 @@ try {
 
   if ($restore) {
     &(Ensure-DotNetSdk) tool restore
+
+    # TODO: Move this to Install-DotNetWorkload.ps1
+    # TODO: Implement the same thing in bash for build.sh, even though we don't build EF.Cocoa on Unix
+    foreach ($workload in $GlobalJson.workloads.PSObject.Properties) {
+      $workloadName = $workload.Name
+      $shortWorkloadName = $workloadName.Substring($workloadName.LastIndexOf(".") + 1)
+
+      $rollbackJson = @{}
+      $rollbackJson[$workloadName] = $workload.Value.version
+
+      $rollbackFile = New-TemporaryFile
```

Comment:
On local Windows build, I'm running into this annoying problem: https://github.com/PowerShell/PowerShell/issues/14100 . Working around it by running `.\eng\build.ps1 -restore` directly from my PowerShell terminal instead of using `Restore.cmd`.